content/administration/maintain/google_oauth.rst

```diff
@@ -0,0 +1,240 @@
+
+.. image:: google_oauth/add-uri.png
+   :class: img-fluid
+   :align: center
+   :alt: Add URI in Creating Credentials for Google OAuth.
+
+- Next, click on :guilabel:`Create` and Google will generate an OAuth :guilabel:`Client ID` and
+  :guilabel:`Client Secret`. Finally, copy each generated value to a Notepad and navigate to the
+  Odoo database.
+
+.. image:: google_oauth/client-credentials.png
+   :class: img-fluid
+   :align: center
+   :alt: Client ID and Client Secret for Google OAuth.
+
+Setup in Odoo
+=============
+
+Enter Google Credentials
+------------------------
+
+First, open Odoo and navigate to the :guilabel:`Apps` module. Then, remove the
+:guilabel:`Apps filter` and type in *Google*. Install the module called :guilabel:`Google Gmail`.
+
+Next, navigate to the :menuselection:`Settings module --> General Settings`, and under the
+:guilabel:`Discuss` section, ensure that the checkbox for :guilabel:`Custom Email Servers` is
+checked. This populates a new option for :guilabel:`Gmail Credentials`. Here, copy and paste the
+:guilabel:`Client ID` and :guilabel:`Client Secret` into the respective fields and :guilabel:`Save`
+the settings.
+
+.. image:: google_oauth/google-credentials.png
+   :align: center
+   :alt: Gmail Credentials in Odoo for Google OAuth.
+
+Configure outgoing email server
+-------------------------------
+
+After that, on the :guilabel:`General Settings` page, under :guilabel:`Custom Email Servers`, click
+:guilabel:`Outgoing Email Servers` to configure the external Gmail account.
+
+.. image:: google_oauth/outgoing-servers.png
+   :class: img-fluid
+   :align: center
+   :alt: Configure Outgoing Email Servers in Odoo.
+
+Then, :guilabel:`create` a new email server and select the option for :guilabel:`Gmail`. Next, fill
+in the :guilabel:`Description` (can be anything) and the email :guilabel:`Username` and click on
+:guilabel:`Connect your Gmail account`.
+
+A new window from :guilabel:`Google` appears to complete the authorization process. Select the
+appropriate email address that is being configured in Odoo.
+
+If the email address is a personal account, an extra step pops up, click :guilabel:`Continue` to
+allow the verification and connect the Gmail account to Odoo.
+
+.. image:: google_oauth/google-unverified.png
+   :class: img-fluid
+   :align: center
+   :alt: Google Hasn't Verified This App.
+
+Finally, allow Odoo to access the Google account by clicking on :guilabel:`Continue` or
+:guilabel:`Allow`.
+
+After that, the page navigates back to the newly configured outgoing email server in Odoo. The
+configuration automatically loads the token in Odoo, and a graphic stating
+:guilabel:`Gmail Token Valid` appears in green.
+
+.. image:: google_oauth/green-token.png
+   :class: img-fluid
+   :align: center
+   :alt: Configure Outgoing Email Servers in Odoo.
+
+Finally, save the settings and :guilabel:`Test the Connection`.
+
```

Comment:
Line 160: delete this line break until Line 159 and 161 are in the same paragraph